examples/offline_inference_vision_language.py

```diff
@@ -114,6 +124,7 @@ def run_minicpmv(question):
     "paligemma": run_paligemma,
     "chameleon": run_chameleon,
     "minicpmv": run_minicpmv,
+    "blip-2": run_blip2,
```

Comment:
```suggestion
    "blip2": run_blip2,
```

Reply:
This value is taken from https://huggingface.co/Salesforce/blip2-opt-2.7b/blob/main/config.json#L8

Reply:
Oh, nevermind then.